src/server/qgshttprequesthandler.cpp

```diff
@@ -218,21 +218,10 @@ void QgsHttpRequestHandler::sendResponse()
   clearBody();
 }
 
-QByteArray QgsHttpRequestHandler::getResponse( const bool returnHeaders,
-    const bool returnBody )
+QPair<QByteArray, QByteArray> QgsHttpRequestHandler::getResponse()
 {
-  if ( ! returnHeaders )
-  {
-    return mResponseBody;
-  }
-  else if ( ! returnBody )
-  {
-    return mResponseHeader;
-  }
-  else
-  {
-    return mResponseHeader.append( mResponseBody );
-  }
+  QPair<QByteArray, QByteArray> response( mResponseHeader, mResponseBody );
```

Comment:
@m-kuhn I'm still working on this PR, I need your help with this return value: I'm using a QPair (and this is